ros_buildfarm/config/release_build_file.py

```diff
@@ -87,8 +87,13 @@ def __init__(self, name, data):  # noqa: D107
                 bool(data['skip_ignored_packages'])
 
         self.sync_package_count = None
+        self.sync_package_percent = None
         self.sync_packages = []
         if 'sync' in data:
+            if 'package_percent' in data['sync']:
+                percent = data['sync']['package_percent']
```

Comment:
Other numeric configurations in this file are getting an explicit conversion. Probably best to maintain the same behavior here.
```suggestion
                percent = float(data['sync']['package_percent'])
```

Reply:
Fixed now.